src/api/config/routes.rb

```diff
@@ -447,6 +447,8 @@ def self.public_or_about_path?(request)
 
     resource :configuration, only: [:show, :update, :schedulers]
 
+    resources :announcements
```

Comment:
@bgeuken, I forgot to add `except: :new`. Could add it? please :pray:

Reply:
Actually I forgot to add this after dropping :new and :edit :laughing: